### ISSUE TITLE
BatchNorm Infinite Loop

### ISSUE THREAD

**Issue by YunseonShin** [PSAL-POSTECH/PyTorchSim#68]
[z4gm.zip](https://github.com/user-attachments/files/17799385/z4gm.zip)

Batchnorm timing simulation does not finish.
